springerlink.md

```diff
@@ -383,13 +369,7 @@ Video content located at link.springer.com/video can generally be operated witho
 
 Video content elsewhere (e.g. chapters or articles) cannot currently be operated with the keyboard. We are actively working to repair this problem. 
 <br><br>
-The SpringerLink checkout process pages are rendered in two separate designs. We are in the process of transferring all content from the older design (located at checkout.springer.com) to the newer design (located at order.springer.com). 
-<br><br>
-At checkout.springer.com, mouse-dependent event handlers are used to display detailed information about sales tax for purchases, although the subtotals and totals are displayed to all users at all times. 
-<br><br>
-Some components on pages located at checkout.springer.com can’t used by keyboard alone (e.g. the “login” and language selection components at the top of each page).
-<br><br>
-All components on pages located at order.springer.com can be operated by keyboard alone. We are in the process of transferring all checkout content to this new design. 
+All components on pages located at order.springer.com can be operated by keyboard alone.
```

Comment:
delete this - already covered in the first sentence, so no need for a separate mention for the checkout as the issues are now resolved 🎉